src/System.IO.IsolatedStorage/tests/project.json

```diff
@@ -32,9 +34,19 @@
     "Microsoft.DotNet.xunit.performance": "1.0.0-alpha-build0040"
   },
   "frameworks": {
-    "netstandard1.7": {}
+    "netstandard1.7": {
+      "imports": [
+        "dotnet5.8"
+      ]
+    },
+    "net463": {}
```

Comment:
I thought you needed the targetting pack in order for net463 to work, is that not the case? I guess perhaps the other dependencies are pulling that in.